lib/meadow_web/resolvers/helpers.ex

```diff
@@ -4,6 +4,13 @@ defmodule MeadowWeb.Resolvers.Helpers do
   """
 
   alias Meadow.Config
+  alias Meadow.Utils.AWS
+
+  def get_presigned_url(_, params, _) do
+    with url <- AWS.presigned_url(Config.upload_bucket(), params) do
+      {:ok, %{url: url}}
+    end
```

Comment:
Would it make sense to handle the error case here as well? Thinking something like:
```suggestion
    case AWS.presigned_url(Config.upload_bucket(), params) do
      {:ok, url} -> {:ok, url}
      {:error, msg} -> {:error, message: "Could not create presigned url", details: msg}
    end
```

Reply:
I get the impulse, but there is a 0% chance that the call to `presigned_url` will fail without something else having failed earlier. It's a completely client-side operation that takes some data and builds a URL.